assets/src/components/activities/common/hints/delivery/HintsDeliveryConnected.tsx

```diff
@@ -43,9 +43,8 @@ const isRequestHintDisabled = (
   correct: boolean,
   graded: boolean,
 ) => {
-  if (!hasMoreHints) return false;
+  if (!hasMoreHints || correct || !correct) return false;
```

Comment:
Similar to the comment above, this is equivalent to  `!hasMoreHints || true` which will always return true